pcsx2/SPU2/Windows/Config.cpp

```diff
@@ -127,9 +126,7 @@ void ReadSettings()
 	dspPluginModule = CfgReadInt(L"DSP PLUGIN", L"ModuleNum", 0);
 	dspPluginEnabled = CfgReadBool(L"DSP PLUGIN", L"Enabled", false);
 
-	// Read WAVEOUT configs:
-	CfgReadStr(L"WAVEOUT", L"Device", Config_WaveOut.Device, L"default");
-	Config_WaveOut.NumBuffers = CfgReadInt(L"WAVEOUT", L"Buffer_Count", 4);
+
 
```

Comment:
Just a minor thing, remove the 2 empty lines here.

Reply:
done :)